### PR TITLE
SOF-1077: made changes to accommodate parts of task

### PR DESCRIPTION
**Pull request by andr9528**
fix: Removed forkSync and forkSynk, which were syncing code with NRK.
fix: Reworked Søren's automerge workflow to not have him as the author of the merge commit.
feature: Added new workflow that merges master to staging.

### CODE REVIEW

**Review by LindvedKrvang — APPROVED** [review]

**Comment by LindvedKrvang** [comment]
Quick reminder that `made changes to accomodate part of task` is not usefull commit message/title.
Your commit messages/titles (and PR titles) should reflect the actual changes that has been done.
So in this case a more applicable name could be `Implement automerge from master to staging and staging to develop` or something like that.

**Review by KvelaGorrrrnio — CHANGES_REQUESTED** [review]
Instead of having two separate workflows that are executed at specific timecodes, it would be better to combine these two into a single workflow, where  the job `merge-staging-to-develop` is dependent on the `merge-master-to-staging` job.
In this case it is very unlikely, but if something delays the first job to past `00.15`, then the changes from master won't propagate to `develop`.

**Review by RasmusAlbrektsen — APPROVED** [review]

**Comment by andr9528** [comment]
@KvelaGorrrrnio The requested changes has been applied. I tested a similar setup in my own testing repository, and it seems like the second merge waited on the first, which is what we wanted.

**Review by KvelaGorrrrnio — APPROVED** [review]
Looks good. Another optimization which is not necessary is to extract the statements, that creates the automerge, into a separate action – [an example](https://github.com/tv2/mediatech-ci-workflows/blob/master/src/github/actions/extract-version/action.yml).

File: .github/workflows/automerge.yml [hunk 1]
```diff
@@ -1,23 +1,43 @@
...
+          git fetch --unshallow
+          git checkout staging
+          git pull
+          git merge --no-ff origin/master -m "Automerge 'master' to 'staging'"
+          git push
+
+  merge-staging-to-develop:
+    name: Merge Staging to Develop
+    runs-on: ubuntu-latest
+    needs: merge-master-to-staging
```
Comment (LindvedKrvang) [inline .github/workflows/automerge.yml:29]: Will this stop the staging to develop merge if there has only been new changes to staging at not master?

**Review by LindvedKrvang — APPROVED** [review]

File: .github/workflows/automerge.yml (hunk 1 above)
Comment (andr9528) [reply .github/workflows/automerge.yml:29]: The 'merge-master-to-staging' still runs, it just says that 'Everything up-to-date', followed by the 'merge-staging-to-develop'. I just tested it on my own repository i set up for precisely testing these kind of things :)

**Pull request merged**